os-jdg7-launch/added/openshift-launch.sh

```diff
@@ -25,8 +25,35 @@ CONFIGURE_SCRIPTS=(
   /opt/run-java/proxy-options
 )
 
+function runServer() {
+  local instanceDir=$1
+
+  echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
+
+  exec $JBOSS_HOME/bin/standalone.sh -c clustered-openshift.xml -bmanagement 127.0.0.1 -Djboss.server.data.dir="$instanceDir" ${JBOSS_HA_ARGS} ${JAVA_PROXY_OPTIONS}
+}
+
+function init_data_dir() {
+  DATA_DIR="$1"
+  if [ -d /tmp/src/data ]; then
+    if [ ! -f "${DATA_DIR}/../data_initialized" ]; then
+      cp -rf /tmp/src/data/* $DATA_DIR
```

Comment:
If we assume the assemble script has done its job then it may be better to copy standalone/data into "${DATA_DIR}", this could also catch any changes an extension may choose to make.

@rcernich Would we support extensions seeding information into the data directory?  I suspect your work allows this but would like to confirm.